tap_hubspot_beta/client_base.py

```
@@ -177,25 +177,28 @@ def curlify_request(self, request):
 
     def validate_response(self, response: requests.Response) -> None:
         """Validate HTTP response."""
-        if 500 <= response.status_code < 600 or response.status_code in [429, 401, 104]:
-            msg = (
-                f"{response.status_code} Server Error: "
-                f"{response.reason} for path: {self.path}"
-            )
+        try:
+            json_response = response.json()
+        except ValueError:
+            json_response = {}
```

I don't know if this has happened before but I think we should also raise a Retriable error if the response can't be parsed to json, this would only hide the error

---

yeah, I think that's alright!